bigquery/src/main/java/org/apache/zeppelin/bigquery/BigQueryInterpreter.java

```diff
@@ -226,19 +226,15 @@ private InterpreterResult executeSql(String sql) {
     String projId = getProperty(PROJECT_ID);
     long wTime = Long.parseLong(getProperty(WAIT_TIME));
     long maxRows = Long.parseLong(getProperty(MAX_ROWS));
-    String sqlDialect = getProperty(SQL_DIALECT, "");
-    Boolean useLegacySql;
-    switch (sqlDialect) {
-      case "standardSQL":
-        useLegacySql = false;
-        break;
-      case "legacySQL":
-        useLegacySql = true;
-        break;
-      default:
-        // Enable query prefix like '#standardSQL' if specified
-        useLegacySql = null;
+    String sqlDialect = getProperty(SQL_DIALECT, "").toLowerCase();
+    // Query prefix like '#standardSQL' can be used in case of null
+    Boolean useLegacySql = null;
+    if (sqlDialect.contains("standard")) {
```

Comment:
I'd prefer exact match - is there a reason to prefer substring/contains?

Reply:
ok, I'd defer to you.